job_bundles/vray_render/template.yaml

```diff
@@ -0,0 +1,76 @@
+specificationVersion: 'jobtemplate-2023-09'
+name: V-Ray Render
+
+parameterDefinitions:
+
+# Render Parameters
+- name: VraySceneFile
+  type: PATH
+  objectType: FILE
+  dataFlow: IN
+  userInterface:
+    control: CHOOSE_INPUT_FILE
+    label: Vray Scene File
+    groupLabel: Render Parameters
+    fileFilters:
+    - label: Vray Scene Files
+      patterns: ["*.vrscene"]
+    - label: All Files
+      patterns: ["*"]
+  description: >
+    Choose the Vray scene file you want to render. Use the 'Job Attachments' tab
+    to add textures and other files that the job needs.
+- name: OutputDir
+  type: PATH
+  objectType: DIRECTORY
+  dataFlow: OUT
+  userInterface:
+    control: CHOOSE_DIRECTORY
+    label: Output Directory
+    groupLabel: Render Parameters
+  default: "./output"
+  description: Choose the render output directory.
+- name: OutputFilename
+  type: STRING
+  userInterface:
+    control: LINE_EDIT
+    label: Output File Name
+    groupLabel: Render Parameters
+  default: "output.png"
+  description: Enter the output filename (with extension of choice).
```

Comment:
Do we support all output file extensions? Or do we need a list here

Reply:
we do not know the list since V-Ray standalone do not have a list. Talked with Mark and we decided to make default the generic file extension and let customer decide based on their need.